src/page/Auth/FindPassword/SendAuthNumber/index.tsx

```diff
@@ -1,34 +1,64 @@
 import { Outlet } from 'react-router-dom';
 import { ReactComponent as KoinLogo } from 'assets/svg/auth/koin-logo.svg';
+import { useState } from 'react';
 import cn from 'utils/ts/className';
+import { useVerifyEmail, useSubmit } from 'query/auth';
 import styles from './SendAuthNumber.module.scss';
 
 export default function FindPassword() {
+  const [emailInput, setEmailInput] = useState('');
+  const [verifyInput, setVerifyInput] = useState('');
+  const [isSendAuth, setIsSendAuth] = useState(false);
+  const verifyEmail = useVerifyEmail(emailInput, setIsSendAuth);
+  const submit = useSubmit(emailInput, verifyInput);
+
   return (
     <>
       <div className={styles.template}>
         <KoinLogo className={styles.logo} />
         <form className={styles.form}>
           <label className={styles.form__label} htmlFor="email">
             이메일 입력
-            <input
-              className={styles.form__input}
-              type="text"
-              id="email"
-            />
-          </label>
-          <label className={styles.form__label} htmlFor="auth-num">
-            인증번호 보내기
             <div className={styles['auth-container']}>
               <input
-                className={cn({ [styles.form__input]: true, [styles['form__input--auth']]: true })}
+                className={cn({
+                  [styles.form__input]: true,
+                  [styles['form__input--auth']]: true,
+                })}
+                onChange={(e) => setEmailInput(e.target.value)}
                 type="text"
-                id="auth-num"
+                id="email"
               />
-              <button type="button" className={styles['auth-button']}>인증번호 발송</button>
+              <button
+                type="button"
+                className={styles['auth-button']}
+                onClick={() => verifyEmail.mutate()}
+                disabled={verifyEmail.isLoading}
+              >
+                {isSendAuth ? '재발송' : '인증번호 발송'}
+              </button>
             </div>
           </label>
-          <button type="button" className={styles.submit}>다음</button>
+          <label className={styles.form__label} htmlFor="auth-num">
+            인증번호 보내기
+            <input
+              className={styles.form__input}
+              type="text"
+              id="auth-num"
+              onChange={(e) => setVerifyInput(e.target.value)}
+            />
+          </label>
+          <button
+            type="submit"
+            onClick={(e) => {
+              e.preventDefault();
+              submit.mutate();
```

Comment:
위 리뷰를 적용하면 아래처럼 쓸 수 있겠죠? 훨씬 직관적이라고 생각해요!
```suggestion
              submit(emailInput, verifyInput);
```